npc/custom/official/lasagna/lasagna_npcs_1.txt

```diff
@@ -0,0 +1,2283 @@
+			mes "So that again proves that I'm the best in this domain!";
+		}
+		next;
+		mes "[Well-known Kittie]";
+		mes "You finished the job on a high note so I'd like to help you in return.";
+		next;
+		mes "[Well-known Kittie]";
+		mes "By the way, do you know what a ^4d4dffSkill Tree^000000 is?";
+		mes "Did I teach you about that?";
+		mes "Do you remember anything about it?";
+		next;
+		mes "[Well-known Kittie]";
+		mes "You can ^4d4dffbring up the Skill Tree by pressing the [Alt+S] shortcut key or clicking the corresponding skill icon.^000000";
+		next;
+		mes "[Well-known Kittie]";
+		mes "Setup the skill you'd like to use so that you can use it while in battle.";
+		next;
+		mes "[Well-known Kittie]";
+		mes "^4d4dffSkills that are activated manually ^000000 can be ^4d4dffplaced on the quickbar^000000 for fast use.";
+		next;
+		mes "[Well-known Kittie]";
+		mes "...Now you know how to use skills.";
+		mes "You should thank me. Nyo-hoho!";
+		completequest 7721;// Cat showoff contest
+		getexp 120,120;
+		doevent "Vigilante Aglio#doram12::OnEvent";
+		close;
+	case 2:
+		mes "[Well-known Kittie]";
+		mes "Do you need to be enlightened again?";
+		next;
+		mes "[Well-known Kittie]";
+		mes "There's nothing special about life in this world.";
+		mes "You'd better make yourself familiar with the skill tree. It is accessed when you click the Skill icon.";
+		next;
+		mes "[Well-known Kittie]";
+		mes "Battle Dorams are specialized in summoning.";
+		mes "Focus on using the summoning skills so that you can win battles.";
+		close;
+	}
+
+OnTouch:
+	if (Class == Job_Summoner && isbegin_quest(7721) == 0)
+		npctalk "Meeow?? Is this another out-of-towner Doram that I haven't met before?";
+	end;
+}
+
+lasa_fild01,102,342,7	script	Vigilante Aglio#doram12	4_DR_AGLIO,{
+	end;
+OnEvent:
+	hideoffnpc "Vigilante Aglio#doram12";
+	sleep2 1000;
+	npctalk "You! You're picking a fight with another innocent passerby?";
+	sleep2 2000;
+	npctalk "Picking a fight? This is what we call training!", "Well-known Troublemaker";
+	sleep2 2000;
+	npctalk "Shut your mouth! You're joining the Vigilante Corp the moment you return to the village, got it?";
+	sleep2 2000;
+	npctalk "Well, uh...I'm...uh...I'm free-spirited! That's all!", "Well-known Troublemaker";
+	sleep2 2000;
+	npctalk "Enough. " + strcharinfo(0) + " the next destination is the tool shop in the village. I'll let you know where it is, again.";
+	sleep2 2000;
+	npctalk "I'll see you again in the village!";
+	navigateto "lasagna",168,124;
+	sleep2 2000;
+	npctalk "You're just full of it.", "Well-known Troublemaker";
+	hideonnpc "Vigilante Aglio#doram12";
+	end;
+OnInit:
+	hideonnpc "Vigilante Aglio#doram12";
+	end;
+}
+
+izlude,117,165,3	script	Cat Paw Shrimp Merchant#iz	4_M_MERCAT1,{
+	npctalk "Here are the shrimps, carrots and catnip!";	// npctalk should be attached to the player
+	switch( select( "Buy", "Sell", "Cancel" ) ) {
+	case 1:
+		callshop "doram_shrimp",1;
```

Comment:
I couldn't reproduce the issue. The script is still workign with @Lemongrass3110 changed